pinot-core/src/main/java/org/apache/pinot/core/common/DataFetcher.java

```diff
@@ -748,6 +756,20 @@ public void readNumValuesMV(int[] docIds, int length, int[] numValuesBuffer) {
       }
     }
 
+    void readBytesValuesMV(int[] docIds, int length, byte[][][] valuesBuffer) {
+      Tracing.activeRecording().setInputDataType(_storedType, _singleValue);
+      if (_dictionary != null) {
+        for (int i = 0; i < length; i++) {
+          int numValues = _reader.getDictIdMV(docIds[i], _reusableMVDictIds, getReaderContext());
```

Comment:
Create reader context before entering the loop. See `readStringValuesMV` for reference

Reply:
Done